src/screens/OtherProfile/index.js

```diff
@@ -406,12 +407,19 @@ const OtherProfile = () => {
     return (
       <>
         <View style={styles.headerImageContainer}>
-          <Image
-            style={styles.profileImage}
-            source={{
-              uri: dataMain.profile_pic_path ?? DEFAULT_PROFILE_PIC_PATH
-            }}
-          />
+          <View style={{marginRight: normalize(22)}}>
+            <CircleGradient
+              fill={dataMain.karma_score ?? 0}
+              size={normalize(100)}
+              width={normalize(2)}>
+              <Image
+                style={styles.profileImage}
+                source={{
+                  uri: dataMain.profile_pic_path ?? DEFAULT_PROFILE_PIC_PATH
+                }}
+              />
+            </CircleGradient>
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `CircleGradient` component is used here to add a gradient circle around the profile picture. However, it seems like there's no error handling for when the `karma_score` is null or undefined. This could potentially lead to unexpected behavior or crashes. Consider adding a default value or some form of error handling.
```diff
            <CircleGradient
-              fill={dataMain.karma_score ?? 0}
+              fill={dataMain.karma_score ?? DEFAULT_KARMA_SCORE}
              size={normalize(100)}
              width={normalize(2)}>
```
Where `DEFAULT_KARMA_SCORE` is a constant that you define with a sensible default value.



<!-- This is an auto-generated comment by OSS CodeRabbit -->